pkg/apiserver/debugapi/endpoint_def.go

```diff
@@ -0,0 +1,521 @@
+		Component: model.NodeKindPD,
+		Path:      "/cluster/status",
+		Method:    endpoint.MethodGet,
+	})
+	c.AddEndpoint(&endpoint.APIModel{
+		ID:        "pd_config_show_all",
+		Component: model.NodeKindPD,
+		Path:      "/config",
+		Method:    endpoint.MethodGet,
+	})
+	c.AddEndpoint(&endpoint.APIModel{
+		ID:        "pd_health",
+		Component: model.NodeKindPD,
+		Path:      "/health",
+		Method:    endpoint.MethodGet,
+	})
+	c.AddEndpoint(&endpoint.APIModel{
+		ID:        "pd_hot_read",
+		Component: model.NodeKindPD,
+		Path:      "/hotspot/regions/read",
+		Method:    endpoint.MethodGet,
+	})
+	c.AddEndpoint(&endpoint.APIModel{
+		ID:        "pd_hot_write",
+		Component: model.NodeKindPD,
+		Path:      "/hotspot/regions/write",
+		Method:    endpoint.MethodGet,
+	})
+	c.AddEndpoint(&endpoint.APIModel{
+		ID:        "pd_hot_stores",
+		Component: model.NodeKindPD,
+		Path:      "/hotspot/stores",
+		Method:    endpoint.MethodGet,
+	})
+	c.AddEndpoint(&endpoint.APIModel{
+		ID:        "pd_labels",
+		Component: model.NodeKindPD,
+		Path:      "/labels",
+		Method:    endpoint.MethodGet,
+	})
+	c.AddEndpoint(&endpoint.APIModel{
+		ID:        "pd_label_stores",
+		Component: model.NodeKindPD,
+		Path:      "/labels/stores",
+		Method:    endpoint.MethodGet,
+		QueryParams: []*endpoint.APIParam{
+			{Model: endpoint.APIParamModelText, Name: "name", Required: true},
+			{Model: endpoint.APIParamModelText, Name: "value", Required: true},
+		},
+	})
+	c.AddEndpoint(&endpoint.APIModel{
+		ID:        "pd_members_show",
+		Component: model.NodeKindPD,
+		Path:      "/members",
+		Method:    endpoint.MethodGet,
+	})
+	c.AddEndpoint(&endpoint.APIModel{
+		ID:        "pd_leader_show",
+		Component: model.NodeKindPD,
+		Path:      "/leader",
+		Method:    endpoint.MethodGet,
+	})
+	c.AddEndpoint(&endpoint.APIModel{
+		ID:        "pd_operator_show",
+		Component: model.NodeKindPD,
+		Path:      "/operators",
+		Method:    endpoint.MethodGet,
+		QueryParams: []*endpoint.APIParam{
+			{Model: endpoint.APIParamModelText, Name: "kind"},
+		},
+	})
+	c.AddEndpoint(&endpoint.APIModel{
+		ID:        "pd_regions",
+		Component: model.NodeKindPD,
+		Path:      "/regions",
+		Method:    endpoint.MethodGet,
+	})
+	c.AddEndpoint(&endpoint.APIModel{
+		ID:        "pd_region_id",
+		Component: model.NodeKindPD,
+		Path:      "/region/id/{regionID}",
+		Method:    endpoint.MethodGet,
+		PathParams: []*endpoint.APIParam{
+			{Model: endpoint.APIParamModelText, Name: "regionID", Required: true},
+		},
+	})
+	c.AddEndpoint(&endpoint.APIModel{
+		ID:        "pd_region_key",
+		Component: model.NodeKindPD,
+		Path:      "/region/key/{regionKey}",
+		Method:    endpoint.MethodGet,
+		PathParams: []*endpoint.APIParam{
+			{Model: endpoint.APIParamModelText, Name: "regionKey", Required: true},
+		},
+	}, endpoint.MiddlewareHandlerFunc(func(req *endpoint.Request) error {
+		req.PathValues.Set("regionKey", url.QueryEscape(req.PathValues.Get("regionKey")))
```

Comment:
Is it possible for them to be a special ParamModel (which inherits Text model while having a middleware)?